native/src/hooks/useClearRouteOnClose.ts

```diff
@@ -4,17 +4,23 @@ import { Dispatch } from 'redux'
 import { RoutePropType, RoutesType } from '../constants/NavigationTypes'
 import { StoreActionType } from '../redux/StoreActionType'
 
-const useClearRouteOnClose = (route: RoutePropType<RoutesType>, dispatch: Dispatch<StoreActionType>): void => {
+const useClearRouteOnClose = (
+  route: RoutePropType<RoutesType>,
+  dispatch: Dispatch<StoreActionType>,
+  clearRouteOnClose = false
```

Comment:
I think this should be a quired property to make clear that this is important. Not clearing by default is weird imo.